src/components/Dashboard/__snapshots__/Dashboard.story.storyshot

```diff
@@ -1568,249 +1579,260 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Dashb
-                                }
-                              }
-                              width=""
-                            >
-                              <button
-                                align="start"
-                                className="table-header-label-start iot--table-head--table-header table-header-sortable bx--table-sort"
-                                data-column="hour"
-                                id="column-hour"
-                                onClick={[Function]}
+                                  <svg
+                                    aria-label="Sort rows by this header in ascending order"
+                                    className="bx--table-sort__icon"
+                                    fill="currentColor"
+                                    focusable="false"
+                                    height={20}
+                                    preserveAspectRatio="xMidYMid meet"
+                                    role="img"
+                                    viewBox="0 0 32 32"
+                                    width={20}
+                                    xmlns="http://www.w3.org/2000/svg"
+                                  >
+                                    <path
+                                      d="M16 4L6 14 7.41 15.41 15 7.83 15 30 17 30 17 7.83 24.59 15.41 26 14 16 4z"
+                                    />
+                                  </svg>
+                                  <svg
+                                    aria-label="Sort rows by this header in ascending order"
+                                    className="bx--table-sort__icon-unsorted"
+                                    fill="currentColor"
+                                    focusable="false"
+                                    height={20}
+                                    preserveAspectRatio="xMidYMid meet"
+                                    role="img"
+                                    viewBox="0 0 32 32"
+                                    width={20}
+                                    xmlns="http://www.w3.org/2000/svg"
+                                  >
+                                    <path
+                                      d="M27.6 20.6L24 24.2 24 4 22 4 22 24.2 18.4 20.6 17 22 23 28 29 22zM9 4L3 10 4.4 11.4 8 7.8 8 28 10 28 10 7.8 13.6 11.4 15 10z"
+                                    />
+                                  </svg>
+                                </button>
+                              </th>
+                              <th
+                                aria-sort="none"
+                                className="table-header-label-start iot--table-head--table-header table-header-sortable"
+                                scope="col"
                                 style={
                                   Object {
-                                    "--table-header-width": "",
+                                    "width": undefined,
```

Comment:
This seems incorrect. Do we have control over this?

Reply:
I'm not sure, I'll look into it

Reply:
Update: The diff is very hard to read here, but if you look above you'll see this is the same as before. Also below you'll see the --table-header-width is still given to the button. I think there was just a small change in DOM structure and it caused the diff to go wild.